Microsoft.WindowsAzure.Storage/tests/cloud_storage_account_test.cpp

```diff
@@ -929,7 +929,7 @@ SUITE(Core)
         std::vector<std::future<void>> results;
         for (int i = 1; i < 0x100; ++i)
         {
-            results.emplace_back(std::async(check_account_permission, i));
+            results.emplace_back(std::async(std::launch::async, check_account_permission, i));
```

Comment:
>std::launch::async [](start = 44, length = 18)

When using `std::launch` instead of `std::deferred`, where does the thread come from?

Reply:
Std library implementation manages a thread pool.